lib/stripe.js

```diff
@@ -2,13 +2,13 @@
 
 const resources = require('./resources');
 
-Stripe.DEFAULT_HOST = 'api.stripe.com';
-Stripe.DEFAULT_PORT = '443';
-Stripe.DEFAULT_BASE_PATH = '/v1/';
-Stripe.DEFAULT_API_VERSION = null;
```

Comment:
These constants I'm moderately worried about removing. Users could previously read/write them, though I don't think that was ever intended. 

They can still read them with `Stripe.getConstant` and all of them have proper ways of writing, eg `new Stripe(key, {host, port, basePath, apiVersion, timeout})`

Reply:
cc @ob-stripe @remi-stripe